hivemq-edge/src/frontend/src/api/hooks/useUnifiedNamespace/useGetUnifiedNamespace.tsx

```diff
@@ -6,8 +6,11 @@ import { ApiError, ISA95ApiBean } from '@/api/__generated__'
 export const useGetUnifiedNamespace = () => {
   const appClient = useHttpClient()
 
-  return useQuery<ISA95ApiBean, ApiError>([QUERY_KEYS.UNIFIED_NAMESPACE], async () => {
-    const item = await appClient.uns.getIsa95()
-    return item
+  return useQuery<ISA95ApiBean, ApiError>({
+    queryKey: [QUERY_KEYS.UNIFIED_NAMESPACE],
+    queryFn: async () => {
+      const item = await appClient.uns.getIsa95()
+      return item
```

Comment:
suggestion(non-blocking): can be returned directly and aligns with other implementation within the PR
```suggestion
      return await appClient.uns.getIsa95()
```